App/Sagas/TextileEventsSagas.ts

```diff
@@ -3,19 +3,15 @@ import { ActionType, getType } from 'typesafe-actions'
 import StorageActions from '../Redux/StorageRedux'
 import {PreferencesSelectors} from '../Redux/PreferencesRedux'
 import AccountActions from '../Redux/AccountRedux'
-import ContactsActions from '../Redux/ContactsRedux'
-import PhotoViewingActions from '../Redux/PhotoViewingRedux'
 import MigrationActions from '../Redux/MigrationRedux'
 import TextileEventsActions from '../Redux/TextileEventsRedux'
 import RNPushNotification from 'react-native-push-notification'
 import { RootAction } from '../Redux/Types'
-import Config from 'react-native-config'
 import Textile, {
-  ContactInfo,
-  ThreadInfo
+  ContactInfo
  } from '@textile/react-native-sdk'
 import { logNewEvent } from './DeviceLogs'
-import { pendingInvitesTask } from './ThreadsSagas'
+import { pendingInvitesTask, cameraRollThreadCreateTask } from './ThreadsSagas'
```

Comment:
moved that logic to it's own function, felt like we crammed it into a specific loop but it shoudl be on it's own, it's not even used in current app ever